Gems/AtomTressFX/Code/Rendering/HairFeatureProcessor.cpp

```diff
@@ -316,6 +322,69 @@ namespace AZ
                 return pass ? true : false;
             }
 
+            bool HairFeatureProcessor::CreateHairParentPass(RPI::RenderPipeline* renderPipeline)
+            {
+                if (HasHairParentPass(renderPipeline))
+                {
```

Comment:
At this point, if the pipeline was already active there might be assets processed.
I suggest stress testing it for switching / changing during active render or else it can come back to us the second people start to use the system

Reply:
"Tests done:
Editor with hair components (both PPLL and Shortcut) and deferred fog, UI editor, material editor, game mode.
Pass hot-reloading in editor."